macrosynergy/pnl/historic_portfolio_volatility.py

```diff
@@ -11,15 +11,18 @@
 sys.path.append(os.getcwd())
```

Comment:
Sorry, I hadn't spotted this one. We should never modify the clients Current Working Directory in my books.

Any reason why @Magnus167 ?

Reply:
These are solely for the initial phase of the project to avoid having to install/re-install between each debug session, as the project has several changes between consecutive review sessions.
Rest assured, these changes will not make it to `develop` (or beyond) thanks to [check_sys_path.sh](https://github.com/macrosynergy/macrosynergy/blob/develop/.github/scripts/check_sys_path.sh) in our unittests